src/FeedIo/Reader.php

```diff
@@ -127,6 +124,24 @@ public function read($url, FeedInterface $feed, \DateTime $modifiedSince = null)
     }
 
     /**
+     * @param  ResponseInterface     $response
+     * @param  FeedInterface         $feed
+     * @return Document
+     */
+    public function handleResponse(ResponseInterface $response, FeedInterface $feed)
```

Comment:
Is there any scenario where this method should be used externally? If not, make it private.